applications/nrf_desktop/README.rst

```diff
@@ -1138,7 +1138,7 @@ The following options are inherited from the ``spi-device`` binding and are comm
 
   .. note::
       To achieve the full speed, data must be propagated through the application and reach Bluetooth LE a few hundred microseconds before the subsequent connection event.
-      If you aim for the lowest latency through the LLPM (a 1-ms interval), the sensor data readout should take no more than 250 us.
+      If you aim for the lowest latency through the LLPM (a 1 ms interval), the sensor data readout should take no more than 250 us.
```

Comment:
Isn't `1 ms` a modifier here, so it needs the hyphen, as opposed to `an interval of 1 ms`?

Reply:
It's technically a modifier, even if the 1 ms is the more important part of that parenthetical, I think.
I'd actually change it to `an interval of 1ms` to make it clearer.